mgmt/rest/tribe_test.go

```diff
@@ -206,6 +208,7 @@ func TestTribeTaskAgreements(t *testing.T) {
 						So(resp.Meta.Code, ShouldEqual, 200)
 						So(len(resp.Body.(*rbody.PluginList).LoadedPlugins), ShouldEqual, 1)
 						pluginToUnload := resp.Body.(*rbody.PluginList).LoadedPlugins[0]
+						<-lpe.pluginAddEvent
```

Comment:
Sorry, I shouldn't have pointed you to this line.  It isn't related to your question below.